Jenkinsfile

```diff
@@ -90,7 +90,6 @@ pipeline {
                         CERT_PASSWORD_FILE_PATH = "C:/BUILD_DEPENDENCIES/third_party_cache/common/certs/certp.txt"
                         COMPONENTS_TO_BUILD = "DockerDriver,Extension,Agent"
 
-                        MSBUILD = "C:/Program Files (x86)/MSBuild/14.0/Bin/MSBuild.exe"
```

Comment:
Not sure about that. Maybe we should leave it here and remove it from ansible?

Reply:
I believe it is better to have this variable set in ansible, where `VisualCPP Build Tools` are installed. 

Imagine someone would like to move MSBuild.exe to another location, consequently we would have to update MSBUILD. It is better to do that in one step when ansible playbook is run. Synchronizing new value of MSBUILD in `jenkinsfile` may be a bit more complicated.

Reply:
no i przekonał!